src/block_library_controller.js

```diff
@@ -319,3 +324,61 @@ BlockLibraryController.prototype.updateButtons = function(savedChanges) {
   var isInLibrary = this.has(blockType);
   this.view.updateButtons(blockType, isInLibrary, savedChanges);
 };
+
+/**
+ * Return a string of all blockTypes for building a library JSON
```

Comment:
Why are you choosing to construct a string of JSON, instead of using a raw JavaScript object / associative array?

(I thought I had a comment on this earlier, but can't find it.)